src/99-appendices/03-hed.md

```diff
@@ -26,54 +26,91 @@ onset	duration	HED
+while the common properties are represented by categorial values appearing in other columns. 
+You may provide a `HED` column and multiple categorical columns to document your events.
+Each of these categorical columns should be documented in a corresponding `*_events.json` sidecar. 
+The column name (e.g., `mycodes`) is the dictionary key to this documentation, as illustrated by the following example.  
 
 Example:
 
 ```JSON
 {
-    "mycodes": {
-        "HED": {
-            "Fixation": "Event/Category/Experimental stimulus, Event/Label/CrossFix, Event/Description/A cross appears at screen center to serve as a fixation point, Sensory presentation/Visual, Item/Object/2D Shape/Cross, Attribute/Visual/Fixation point, Attribute/Visual/Rendering type/Screen, Attribute/Location/Screen/Center",
-            "Target": "Event/Label/Target image, Event/Description/A white airplane as the RSVP target superimposed on a satellite image is displayed., Event/Category/Experimental stimulus, (Item/Object/Vehicle/Aircraft/Airplane, Participant/Effect/Cognitive/Target, Sensory presentation/Visual/Rendering type/Screen/2D), (Item/Natural scene/Arial/Satellite, Sensory presentation/Visual/Rendering type/Screen/2D)",
-            "Button": "..."
+   "mycodes": {
+       "LongName": "Local event type names",
+       "Description": "Main types of events that comprise a trial",
+       "Levels": {
+          "Fixation": "Fixation cross is displayed",
+          "Target":   "Target image appears",
+          "Button":   "Subject presses a button"
+       },	  
+       "HED": {
+           "Fixation": "Event/Category/Experimental stimulus, Event/Label/CrossFix,
+		       Event/Description/A cross appears at screen center to serve as a fixation point,
+		       Sensory presentation/Visual, Item/Object/2D Shape/Cross,
+		       Attribute/Visual/Fixation point, Attribute/Visual/Rendering type/Screen,
+		       Attribute/Location/Screen/Center",
+           "Target":   "Event/Label/TargetImage, Event/Category/Experimental stimulus,
+		       Event/Description/A white airplane as the RSVP target superimposed on a satellite image is displayed.,
+		       Item/Object/Vehicle/Aircraft/Airplane, Participant/Effect/Cognitive/Target,
+		       Sensory presentation/Visual/Rendering type/Screen/2D),
+		       (Item/Natural scene/Arial/Satellite,
+		       Sensory presentation/Visual/Rendering type/Screen/2D)",
+           "Button":   "Event/Category/Participant response, Event/Label/PressButton, 
+		       Event/Description/The participant presses the button as soon as the target is visible,
+		       Action/Button press"
         }
-    }
+   }
 }
 ```
+Downstream tools should not distinguish between tags specified using the explicit HED column and 
+the categorical specifications, but should form the union before analysis. Further,
+the normal BIDS inheritance principle applies, so the data dictionaries can
+appear higher in the BIDS hierarchy.  
```

Comment:
```suggestion
the categorical specifications, but should form the union before analysis.
Further, the [inheritance principle](../02-common-principles.md#the-inheritance-principle) applies,
so the data dictionaries can appear higher in the BIDS hierarchy.  
```

Reply:
Corrected, but again, I am not sure about the rules that should be followed for inserting new lines and the extra spaces after periods.